au/code/au/zero_test.cc

```diff
@@ -54,20 +55,20 @@ TEST(WrappedInt, BasicInterfaceWorksAsExpected) {
 
 TEST(Zero, MinusZeroIsZero) {
     constexpr auto zero_minus_zero = ZERO - ZERO;
-    EXPECT_EQ(ZERO, zero_minus_zero);
+    EXPECT_THAT(zero_minus_zero, Eq(ZERO));
 }
 
 TEST(Zero, PlusZeroIsZero) {
     constexpr auto zero_plus_zero = ZERO + ZERO;
-    EXPECT_EQ(ZERO, zero_plus_zero);
+    EXPECT_THAT(zero_plus_zero, Eq(ZERO));
 }
 
 TEST(Zero, ComparableToArbitraryQuantities) {
-    EXPECT_EQ(ZERO, WrappedInt{0});
+    EXPECT_THAT(ZERO, Eq(WrappedInt{0}));
     EXPECT_LT(ZERO, WrappedInt{1});
     EXPECT_GT(ZERO, WrappedInt{-1});
 
-    EXPECT_EQ(ZERO, WrappedInt{0});
+    EXPECT_THAT(WrappedInt{0}, Eq(ZERO));
```

Comment:
Not sure why these are repeated, but I like that you're making them distinct by swapping the order of the args :sweat_smile:

Reply:
I dug into it a bit more.  Came in with c86431a8aaf770bcf13157a1cc6f5436625b42cf.  I'm thinking that it was because the test immediately below it had a "true" block and a "false" block.  I'm thinking maybe we just delete the redundant tests?

Reply:
Honestly, yeah, that's probably the best approach.